src/main/java/edu/kis/powp/jobs2d/drivers/command/ComplexCommand.java

```diff
@@ -0,0 +1,8 @@
+package edu.kis.powp.jobs2d.drivers.command;
+
+public class ComplexCommand {
```

Comment:
Complex command should be implemented in driver commands hierarchy, i.e., it should implement DriverCommand interface